astro/src/components/api/APIField.astro

```diff
@@ -14,9 +14,10 @@ interface Props {
   deprecated?: boolean;
   renderif?: boolean;
   readonly?: boolean;
+  show_since?: boolean;
 }
 
-const { name, description, type, required, optional, since, defaults, deprecated, renderif=true, readonly }: Props = Astro.props;
+const { name, description, type, required, optional, since, defaults, deprecated, renderif=true, show_since=true, readonly }: Props = Astro.props;
```

Comment:
Are we ever using `show_since=false`? I didn't see that being passed in anywhere. If not, we should remove this until we need it.

Reply:
Yes. We consume it on line 31 and set/provide it on `_userinfo-response`